src/Components/SystemDetail/RecommendationsTable.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { Table, TableHeader, TableBody, expandable } from '@patternfly/react-table';
+import propTypes from 'prop-types';
+import { flatMap } from 'lodash';
+import { EmptyTable } from '@redhat-cloud-services/frontend-components/EmptyTable';
+import { EmptyStateDisplay } from '../EmptyStateDisplay/EmptyStateDisplay';
+import { CheckCircleIcon } from '@patternfly/react-icons';
+import { TextContent, Text, TextVariants } from '@patternfly/react-core';
+import successColor from '@patternfly/react-tokens/dist/esm/global_success_color_100';
```

Comment:
@karelhala -
I would like to know whether this is a correct way to import color for icon or not.

Reply:
Modified import statement.